labjack_provider/src/peripherals/dc_drive.cpp

```diff
@@ -0,0 +1,105 @@
+
+#include "dc_drive.hpp"
+
+//Instantiate the DC_Driver with the analogs that create the forward drive, reverse drice, and potentiometer to measure angle. 
+
+//Including setup with angle (Don't know if there is going to be a default)
+
+DC_Drive::DC_Drive(const char* forwardName, const char* reverseName, const char* potentiometerName, double angle): Peripheral() {
+    this->forwardName = forwardName;
+    this->reverseName = reverseName;
+    this->potentiometerName=potentiometerName;
+    this->angle = angle;

```

Comment:
How can we trust what angle we are at simply by being told what angle we are at. This should instead read the potentiometer pin's value and convert to a valid angle to set this value for the object.